### ISSUE TITLE
Question about the experimental setup with VIolent-Flows Dataset in the paper "Efficient Violence Detection Using 3D Convolutional Neural Networks"

### ISSUE THREAD

**Issue by NguyenDucLamK63** [JimLee1996/AVSS2019#13]
First of all, I would like to thank you for your contribution to the community with your great work.
I'm really interested in your study and I am trying to implement your code with the same experimental setup with VIolent-Flows Dataset like in your paper. However there are something ambiguous for me here :
    +, The learning rate and Batchsize in the paper in VIolent-Flows Dataset  is 10^-4 and 16. I followed the Experiment Setup of paper but the result was only "94.35±2.28% " compared to the result of paper is : "97.17±0.95%". If I use learning rate = 10^-3 and Batchsize = 32 the result will be :" 96,768 ± 2.38%", is approximate to the result of paper. So I wonder if the Experiment Setup of paper is correct? Can you help me check it again? 
Thank you!
